docs/contributing/docs_style_guide.md

```diff
@@ -12,7 +12,7 @@
    -- For file name, use underscores between words and prefix all file names with the page name, e.g. context_display_hints.jpg for the image showing how to set display hints in the context menu.
 - Use `!!! note` at the start of a paragraph to have it rendered as a note like this:
```

Comment:
Maybe this should specify that you need to have a title, as it appears required. Also might be good to explictly display the note below.
> Use `!!! "Note Title"` at the start of a paragraph to have it rendered as a note like this:
> ```
> !!! "Helpful Tip" I am a helpful tip!
> ```

Reply:
Good idea. Done!